<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FramePQGamma</class>
 <widget class="QFrame" name="FramePQGamma">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutY_2" stretch="0,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutX">
       <item>
        <widget class="QSpinBox" name="spinBoxX">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hSliderX">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelCurve">
       <property name="font">
        <font>
         <pointsize>150</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;R&lt;/span&gt;&lt;span style=&quot; color:#00ff00;&quot;&gt;G&lt;/span&gt;&lt;span style=&quot; color:#0000ff;&quot;&gt;B&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayoutY">
     <item>
      <widget class="QSpinBox" name="spinBoxY">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximum">
        <number>4095</number>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QSlider" name="vSliderY">
       <property name="maximum">
        <number>4095</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutShow">
       <item>
        <widget class="QCheckBox" name="checkBoxGamma">
         <property name="text">
          <string>001F</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxR">
         <property name="text">
          <string>R</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxG">
         <property name="text">
          <string>G</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxB">
         <property name="text">
          <string>B</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutCtrl">
       <item alignment="Qt::AlignmentFlag::AlignRight">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>操控目标：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton1">
         <property name="text">
          <string>R</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton2">
         <property name="text">
          <string>G</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton3">
         <property name="text">
          <string>B</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutRW">
       <item>
        <widget class="QPushButton" name="pushButtonRead">
         <property name="text">
          <string>Read</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonWrite">
         <property name="text">
          <string>Write</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLoad">
         <property name="text">
          <string>Load ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave">
         <property name="text">
          <string>Save ...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
